pika_psql.go

```diff
@@ -254,19 +258,24 @@ func (b *basePsql[T]) ClearAll() QuerySet[T] {
 }
 
 // Create creates a new record in the database.
-func (b *basePsql[T]) Create(x *T) error {
+func (b *basePsql[T]) Create(x *T, options ...PikaOption) error {
 	if b.err != nil {
 		return b.err
 	}
 
 	origIgnoreOrderBy := b.ignoreOrderBy
 	b.ignoreOrderBy = true
-	q, args := b.CreateQuery(x)
+
+	q, args := b.CreateQuery(x, options...)
 	b.ignoreOrderBy = origIgnoreOrderBy
 
 	// Execute query
 	err := b.psql.Queryable().Get(x, q, args...)
 	if err != nil {
+		// ignore no rows in resultset error when ignoreConflict is set to true, this is a normal case
+		if errors.Is(err, sql.ErrNoRows) && insertOnConflict(options...) {
```

Comment:
Just a suggestion, maybe you could do something like this:

```golang
if errors.Is(err, sql.ErrNoRows) && getOptions(opts...)&InsertOnConflictionDoNothing > 0 {

...

func getOptions(opts ...PikaOption) PikaOption {
	o := 0
	if opts == nil {
		return o
	}
	for _, opt := range opts {
		o = o | opt
	}

	return o
}
```

You dont need to change this, just thinking, this could make it easier when (if) theres lots of options, since we dont need a dedicated function to return each option.